pymodbus/pdu/pdu.py

```diff
@@ -118,6 +118,20 @@ class ExceptionResponse(ModbusPDU):
     GATEWAY_PATH_UNAVIABLE = 0x0A
     GATEWAY_NO_RESPONSE = 0x0B
 
+    exception_code_dict = {
+        0x00: "None",
```

Comment:
Please use the constants defined above, to avoid mismatch between constant and value.

Reply:
This is a constant meaning the name should be capitalized